README.md

```diff
@@ -23,245 +19,22 @@ institutions in the [Dutch Digital Heritage Network](https://netwerkdigitaalerfg
-bin/run.js sources:query --uris https://query.wikidata.org/sparql#entities-all --query Rembrandt
-```
-
-Add `--loglevel` to the commands to see what's going on underneath. For example:
-
-    bin/run.js sources:query --uris https://data.cultureelerfgoed.nl/PoolParty/sparql/term/id/cht --query fiets --loglevel info
-
-Search results are piped to stdout. Redirect these elsewhere for further analysis. For example:
-
-    bin/run.js sources:query --uris https://data.cultureelerfgoed.nl/PoolParty/sparql/term/id/cht --query fiets > cht.txt
-
-## Query sources via GraphQL
-
-### Start server
-
-    docker-compose up --build
-
-### Open the GraphQL editor
-
-http://localhost:3123/graphiql
-
-### List queryable sources
-
-```graphql
-query Sources {
-  sources {
-    uri
-    name
-    alternateName
-    creators {
-      uri
-      name
-      alternateName
-    }
-  }
-}
-```
-
-### Query one or more sources for terms
-
-#### Query a single source
-
-```graphql
-# Query Cultuurhistorische Thesaurus (CHT)
-query {
-  terms(
-    sources: ["https://data.cultureelerfgoed.nl/PoolParty/sparql/term/id/cht"],
-    query: "fiets",
-    queryMode: OPTIMIZED   # Forwards-compatible query type.
-  ) {
-    source {
-      uri
-      name
-      creators {
-        uri
-        name
-        alternateName
-      }
-    }
-    result {
-      __typename
-      ... on Terms {
-        terms {
-          uri
-          prefLabel
-          altLabel
-          hiddenLabel
-          scopeNote
-          seeAlso
-          broader {
-            uri
-            prefLabel
-          }
-          narrower {
-            uri
-            prefLabel
-          }
-          related {
-            uri
-            prefLabel
-          }
-        }
-      }
-      ... on Error {
-        message
-      }
-    }
-  }
-}
-```
+### Packages
 
-#### Query multiple sources
+This repository contains the following packages:
 
-```graphql
-# Query RKDartists and NTA simultaneously
-query {
-  terms(
-    sources: ["https://data.netwerkdigitaalerfgoed.nl/rkd/rkdartists/sparql", "http://data.bibliotheken.nl/thesp/sparql"],
-    query: "Gogh",
-    queryMode: OPTIMIZED   # Forwards-compatible query type.
-  ) {
-    source {
-      uri
-      name
-      creators {
-        uri
-        name
-        alternateName
-      }
-    }
-    result {
-      __typename
-      ... on Terms {
-        terms {
-          uri
-          prefLabel
-          altLabel
-          hiddenLabel
-          scopeNote
-          seeAlso
-        }
-      }
-      ... on Error {
-        message
-      }
-    }
-  }
-}
-```
+- [network-of-terms-catalog](packages/network-of-terms-catalog): the catalog of terminology sources in the Network of
+  Terms that can be queried;
+- [network-of-terms-cli](packages/network-of-terms-cli): query the Network of Terms from the command line;
+- [network-of-terms-graphql](packages/network-of-terms-graphql): a GraphQL query API to find terms;
```

Comment:
Very minor: we could omit 'query' here - 'find terms' already makes clear this is about querying?